src/kirby/components/list/list-item-color.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, Input, Renderer2, ElementRef, OnChanges, SimpleChanges } from '@angular/core';
```

Comment:
There is a `directive` subfolder, in the list component folder. Please move it to there.

Reply:
Added it to directives folder